### ISSUE TITLE
Cannot find a library instance of LibDBIcon-1.0

### ISSUE THREAD

**Issue by TomCats** [syndenbock/MinimapButtonButton#26]
I'm seeing this error in retail with your addon version 1.17.1 in the current 10.1.5 live retail version of WoW when trying to use it with any addon that uses LibDBIcon.  No issues with it interfacing with my addon TomCat's Tours, however.

Message: Interface/AddOns/MinimapButtonButton/Logic/Main.lua:84: Cannot find a library instance of "LibDBIcon-1.0".
Time: Wed Jul 12 11:40:08 2023
Count: 1
Stack: Interface/AddOns/MinimapButtonButton/Logic/Main.lua:84: Cannot find a library instance of "LibDBIcon-1.0".
[string "=[C]"]: in function `error'
[string "@Interface/AddOns/AngryKeystones/libs/LibStub/LibStub.lua"]:38: in function `LibStub'
[string "@Interface/AddOns/MinimapButtonButton/Logic/Main.lua"]:84: in function <Interface/AddOns/MinimapButtonButton/Logic/Main.lua:83>
[string "@Interface/AddOns/MinimapButtonButton/Logic/Main.lua"]:98: in function <Interface/AddOns/MinimapButtonButton/Logic/Main.lua:97>
[string "@Interface/AddOns/MinimapButtonButton/Logic/Main.lua"]:221: in function <Interface/AddOns/MinimapButtonButton/Logic/Main.lua:218>
[string "@Interface/AddOns/MinimapButtonButton/Logic/Main.lua"]:400: in function <Interface/AddOns/MinimapButtonButton/Logic/Main.lua:396>

Locals: (*temporary) = "Cannot find a library instance of "LibDBIcon-1.0"."

**Comment by TomCats** [comment]
This is more of an FYI as it is probably an issue with the other addons, but should still not cause your addon to throw an errors if something is wrong with theirs if it can be helped.  Installing an addon like BugGrabber/BugSack that probably has a more recent LibDBIcon seems to help (and which is written by the same author as LibDBIcon)

**Comment by syndenbock** [comment]
Looking at the message I think it's not even an error itself, it's just me requesting LibDBIcon-1.0 from LibStub and the library not being available. I just assumed in that case LibStub would return nil but re-checking the docs I need to pass a silent flag for LibStub to not throw an error.
So this should be fixed as soon as EU servers come back up so I can test it :D

**Comment by TomCats** [comment]
That makes sense.  Also PTR is usually up when the live realms are down in case that's of use to you

**Comment by syndenbock** [comment]
Yup, seems fixed by passing the silent flag to LibStub, version 1.17.2 is being uploaded this moment.
